doc/2/guides/essentials/security/index.md

```diff
@@ -107,7 +107,32 @@ curl -X GET 'http://localhost:7512/?pretty'
 
 ## Defining Profiles
 
-A `profile` definition is a JSON object that contains an array of policies, each composed of a roleId and an array of restrictions:
+A `profile` definition is a JSON object that contains an optional rate limit parameter, and an array of policies.
+
+### Rate limit
+
+<SinceBadge version="2.0.3" />
+
+The rate limit parameter controls how many API requests a user can send, **per second and per node**. Further requests made by a user that exceed the limit are rejected with a `429 Too Many Requests` error.  
+
+If no rate limit is defined, or if it is set to 0, then no limit is applied.
+
+If a user has multiple profiles with rate limits, the most permissive limit applies.
```

Comment:
```suggestion
If a user has several profiles with rate limits, the most permissive limit applies.
```
And, is the most permissive overriding the rest and not the most restrictive ?

Reply:
Suggestion applied.

To answer your question: because in a whitelist strategy, no rights are granted by default, and roles/profiles give only the necessary rights. 
Following that line of thought, applying rate limits is, in fact, the same as giving the right to send a certain amount of requests per second.
So, for example, if a user has 2 profiles with rate limits of 200 and 50, then this is the same as  giving the right to send 200 reqs/s and the right to send 50 reqs/s. 
So I think it makes better sense to use the most permissive limit.

Reply:
Last question: why applying this limit to the profiles and not directly to the users ?

Reply:
Because in this way we can apply rate limits on group of users instead of individual users